### ISSUE TITLE
Can't play any video, the link could be blocked by Youtube

### ISSUE THREAD

**Issue by Kopisusu10** [TeamNewPipe/NewPipe#7759]
### Checklist

- [x] I am using the latest version - x.xx.x 
- [ ] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [ ] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [ ] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Device info

 - Android version/Custom ROM version:
 - Device model:

**Comment by mhmdanas** [comment]
Duplicate of #7734. Also, please don't remove the template next time you remove an issue.

**Comment by xMxster** [comment]
Yeah. Seems like something broke for all of us.

I advice you to close this, and wait for the update.

I suggest you to download F-Droid and use NewPipe Legacy meanwhile.
